getting-started/cluster/hybrid-clusters.md

```diff
@@ -0,0 +1,27 @@
+# Deploying to Hybrid Linux/Winodws K8s Clusters
```

Comment:
I think this file needs to be under /howto.
@orizohar how do you think?

cc/ @msfussell @yaron2

Reply:
Makes sense, moved to howtos/